CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+# 1.2.10 (2021-05-27)
+## Edge Agent
+### Bug Fixes
+* Flatten additional properties of SystemInfo metrics ( [12b52ba](https://github.com/Azure/iotedge/commit/12b52babe6fba24b136971a5d1ce1b8df387b1d7) )
```

Comment:
Please mention the base image update:
`* Update Base Images to address gzip vulnerability [CVE-2022-23267](https://msrc.microsoft.com/update-guide/vulnerability/CVE-2022-23267) [CVE-2022-29117](https://msrc.microsoft.com/update-guide/vulnerability/CVE-2022-29117)`

Reply:
Sorry the description of this is wrong for these links it should have been `.NET and Visual Studio Denial of Service Vulnerability`